binding/Binding/GRContext.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace SkiaSharp
 {
-	public unsafe class GRContext : SKObject, ISKReferenceCounted
+	public unsafe class GRContext : SKObject, ISKReferenceCounted, ISKSkipObjectRegistration
```

Comment:
To mark an object as "do not add to dictionary", add the `ISKSkipObjectRegistration` interface.
 - `GetInstance` will always return `null`
 - `GetOrAddObject` will always return a new object
 - `RegisterHandle` is a no-op
 - `DeregisterHandle` is a no-op